workflow/envs/filter_reads.yaml

```
@@ -0,0 +1,5 @@
+channels:
+  - conda-forge
+dependencies:
+  - rust-script>=0.17.0
+  - rust
```

maybe try
```suggestion
  - rust
  - compiler-c
```
here to get a `${CC}` which `bzip2-sys`'s build can use.